app/src/test/java/pm/gnosis/heimdall/ui/safe/SafeOverviewViewModelTest.kt

```diff
@@ -140,6 +142,36 @@ class SafeOverviewViewModelTest {
                 .assertError(error)
     }
 
+    @Test
+    fun loadSafeInfoLoadsFromCache() {
+        val testObserver = TestObserver.create<SafeInfo>()
+        val safeInfo = SafeInfo("0x0", Wei(BigInteger.ZERO), 0, emptyList(), false)
+        given(repositoryMock.loadInfo(MockUtils.any())).willReturn(Observable.just(safeInfo))
+
+        viewModel.loadSafeInfo(BigInteger.ZERO).subscribe(testObserver)
+
+        then(repositoryMock).should().loadInfo(BigInteger.ZERO)
```

Comment:
These should probably be at the end of the test, else they might be triggered again by the second call